src/plugins/data/common/field_formats/types.ts

```diff
@@ -25,6 +25,7 @@ export type FieldFormatsContentType = 'html' | 'text';
 /** @internal **/
 export interface HtmlContextTypeOptions {
   field?: any;
+  indexPattern?: any;
```

Comment:
Is it possible to remove `any` from both of these?

Reply:
Good idea but I'd prefer to do it in a new branch. https://github.com/elastic/kibana/issues/75620 - for tracking.